.buildconfig-android.yml

```diff
@@ -99,6 +99,13 @@ projects:
     - name: nimbus
       type: aar
     description: The Nimbus SDK, compiled to work with the Application Services full-megazord.
+  merinoclient:
```

Comment:
Most of the first commit is boilerplate from [this wonderful guide](https://mozilla.github.io/application-services/book/howtos/adding-a-new-component.html) for adding a new component.